Source/Core/DolphinQt/Config/Mapping/IOWindow.cpp

```diff
@@ -563,6 +592,16 @@ void IOWindow::UpdateExpression(std::string new_expression, UpdateMode mode)
   const auto error = m_reference->SetExpression(std::move(new_expression));
   const auto status = m_reference->GetParseStatus();
   m_controller->UpdateSingleControlReference(g_controller_interface, m_reference);
+  
+  // This is the only place where we need to update the shown user variables:
+  while (m_variables_combo->count() > 4)  // Keep the previous items
```

Comment:
We generally prefer comments either above the line or within the statement, but not on the side, since they can make future refactoring a tiny bit more annoying if the line ever changes.

Reply:
The only comments I put on the right are generally "extra" comments that we could do without. That comment isn't worth taking a line of its own IMO.

Reply:
removed all comments on the right of code.

Reply:
If this is an actual rule, it should be here:
https://github.com/dolphin-emu/dolphin/blob/master/Contributing.md
but I think it's good to allow it in some cases (like this)